pkg/vere/lord.c

```diff
@@ -1158,7 +1158,7 @@ u3_lord_init(c3_c* pax_c, c3_w wag_w, c3_d key_d[4], u3_lord_cb cb_u)
   //  spawn new process and connect to it
   //
   {
-    c3_c* arg_c[10];
+    c3_c* arg_c[9];
```

Comment:
Everything else looks self explanatory except the change here (just because I don't know what it's for)

Reply:
The Windows build used 9 arguments and therefore needed a 10-element array since the array needs to be NULL-terminated. Linux and macOS builds only need 8 arguments and therefore only need a 9-element array.